src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -1302,69 +1332,71 @@ ActionResult InstrumentClipView::padAction(int32_t x, int32_t y, int32_t velocit
 				if (afh == NULL) {
 					continue;
 				}
-
-				static int32_t MaxFiles = 25;
-				String fnArray[MaxFiles];
-				char const* currentPathChars = afh->filePath.get();
-				if (currentPathChars == NULL) {
+				char const* path = afh->filePath.get();
+				if (path == NULL) {
```

Comment:
`path` will never be null, but it might be `&nothing`. Can probably skip this null check entirely, if it crashes something the blackbox will tell us pretty immediately.

Reply:
Changed to check for &nothing. 

Leaving the other null check in as it is not expensive, not in the hot loop, and the whole function is only triggered by user interaction, i.e. will not run very often in the grand scheme of things.